build.sbt

```diff
@@ -26,7 +26,7 @@ val datastoreVersion = "1.6.3"
 val guavaVersion = "28.2-jre"
 val jacksonVersion = "2.10.2"
 val jodaTimeVersion = "2.10.5"
-val protobufVersion = "3.10.0"
+val protobufVersion = "3.11.1" // needs to match protoc-jar version in project/plugins.sbt
```

Comment:
It shouldn't have to match if you depend on `protoc-jar` only for compiling `.proto` and not its transitive `protobuf-java` dependency.